api/scpca_portal/models/dataset.py

```diff
@@ -263,6 +272,62 @@ def original_file_zip_paths(self) -> Set[Path]:
 
         return original_file_zip_paths
 
+    @property
+    def original_file_zip_paths_new_structure(self) -> Set[Path]:
```

Comment:
- Let's move anything that doesn't need to know about the dataset back to the original file.
- Instead of "new_structure" just append "dataset" to different versions of the same function if you need to. (i.e `original_file_zip_paths_dataset`). Prepending would also be fine.
- I imagine that this could be made more simple by only checking if it is merged and then getting the attribute off of the original_file based on what the dataset is asking for.